pkg/server/api/svid/v1/server.go

```diff
@@ -54,8 +55,39 @@ func (s server) MintX509SVID(ctx context.Context, req *svid.MintX509SVIDRequest)
 	}, nil
 }
 
-func (s server) MintJWTSVID(context.Context, *svid.MintJWTSVIDRequest) (*svid.MintJWTSVIDResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "not implemented")
+func (s server) MintJWTSVID(ctx context.Context, req *svid.MintJWTSVIDRequest) (*svid.MintJWTSVIDResponse, error) {
```

Comment:
I'm going to add some stuff in the middleware and rpccontext packages as part of authorization that handlers can invoke as a light assertion that the middleware was configured properly. The middleware will still take care of the heavy lifting around the authorization but will add stuff to the context.